inference/python/spec_infer.py

```diff
@@ -148,8 +148,6 @@ def main():
         results = llm.generate(prompts)
     else:
         result = llm.generate("Three tips for staying healthy are: ")
-        
-    llm.stop_server()
```

Comment:
any reason why we need to remove this? @april-yyt

Reply:
That's a misdeletion, I will add it back.